components/Article/EditLink/index.tsx

```diff
@@ -0,0 +1,42 @@
+import React from 'react';
+import { FormattedMessage } from 'react-intl';
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
+import { faPencil } from '@fortawesome/free-solid-svg-icons';
+import styles from './index.module.scss';
+import { useLocale } from './../../../hooks/useLocale';
+
+interface Props {
+  relativePath?: string;
+  editPath?: string;
+  absolutePath?: string;
+}
+
+// TODO(HinataKah0): Change this
+const baseURL =
+  'https://github.com/nodejs/nodejs.org/edit/major/website-redesign';
+
+const EditLink = ({ relativePath, editPath, absolutePath }: Props) => {
+  const { currentLocale } = useLocale();
+
+  if (!relativePath && !editPath && !absolutePath) return null;
+
+  const href =
+    absolutePath ||
+    (relativePath
+      ? `${baseURL}/pages/${currentLocale.code}/${relativePath}`
```

Comment:
Need input for this.

Previous URL is `/content/learn/` and new URL is `/pages/{2-letters-language-code}`. Currently I am using `currentLocale`, not sure if it is correct.

Reply:
I believe it should be `{2-letters-language-code}/relativePath`

Reply:
Sorry my comment was a bit shallow & unclear. But my intention was similar to the discussion below about the redirection behavior.